packages/cinterop/build.gradle.kts

```diff
@@ -470,6 +470,11 @@ fun getSharedCMakeFlags(buildType: BuildType, ccache: Boolean = true): Array<Str
         add("-DREALM_NO_TESTS=1")
         add("-DREALM_BUILD_LIB_ONLY=true")
         add("-DREALM_CORE_SUBMODULE_BUILD=true")
+        // This will prevent exporting Core's symbols. which is useful when combining for example Swift SDK and Kotlin Multiplatform
```

Comment:
```suggestion
        // This will prevent exporting Core's symbols which is useful when combining for example the Swift SDK and Kotlin Multiplatform
```